python/tests/test_aggregates.py

```diff
@@ -0,0 +1,94 @@
+import unittest
+from feldera import SQLContext
+from tests import TEST_CLIENT
+
+class TestAggregatesBase(unittest.TestCase):
+    def execute_query(self, data, expected_data, pipeline_name, table_name, view_name, view_query):
+        sql = SQLContext(f'{pipeline_name}', TEST_CLIENT).get_or_create()
+        
+        sql.register_table_from_sql(f''' 
+                                    CREATE TABLE {table_name}(
+                                        id INT, c1 TINYINT, c2 TINYINT NOT NULL, c3 INT2, c4 INT2 NOT NULL, c5 INT, c6 INT NOT NULL,c7 BIGINT,c8 BIGINT NOT NULL);''')
+        sql.register_materialized_view(f"{view_name}", f"{view_query}")
+        
+        out = sql.listen(f"{view_name}")
+        sql.start()
+        sql.input_json(f"{table_name}", data)
+        sql.wait_for_completion(True)
+        out_data = out.to_dict()
+        print(out_data)
+        for datum in expected_data:
+            datum.update({"insert_delete": 1})
+        assert expected_data == out_data
+
+class Avg(TestAggregatesBase):
+    def test_avg_value(self):
+        pipeline_name = "test_avg_value"
+        data = [
```

Comment:
can we also move the data to the base class? Let's have some default data there.
Let's also add a method "add_data" to the base class which can be used to add additional data.
The add_data method should have a boolean insert/delete argument, so we can also delete data if needed.